deriv.js

```diff
@@ -182,9 +182,9 @@ function handleCodeCopyClick() {
     code_text = code_box.children[0].textContent.trim();
   } else {
     code_box = code_box.nextElementSibling;
-    if (code_box.children.length === 14) {
+    if (code_box.children.length === 4) {
```

Comment:
Same here. A variable would be nice to understand what the 4 children represent

Reply:
Replaced with a variable.